content/blog/Why-I-wanted-to-help-translating-the-AlmaLinux-website-to-Hebrew.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Why I wanted to help translate the AlmaLinux website to Hebrew?"
+type: blog
+author: 
+ name: "Noam Alum"
+ bio: "System Administrator at JetServer Hosting"
+ image: /users/noam.jpeg
+date: '2024-04-10'
```

Comment:
You will want to update this date whenever it's slated for publication; otherwise, it will be published below existing/old blog posts :).

Reply:
Yup! I'll update it when I get it scheduled. :D